lib/i18n/american-english.js

```diff
@@ -327,7 +327,7 @@ function isGoodWord(word) {
     // filter out words that cannot be in the dataset,
     // because they would be either tokenized/preprocessed out or
     // they are unlikely to be used with voice
-    return /^([a-zA-Z0-9-][a-zA-Z0-9.-]*|'s|,|\?)$/.test(word);
+    return /^([a-zA-Z0-9-][a-zA-Z0-9.-]*|'s|,|\?|!|\.)$/.test(word);
```

Comment:
This unfortunately defeats the point of this check, which is to avoid a confusion where a sentence ends with a "." whether the "." is part of the parameter or not. (In voice, pretty much everything ends with "." because the STT will fix punctuation)

Reply:
fixed (this should have been removed since I switched to using cleanParameters that contain no punctuation marks)